app/views/layouts/_textual_groups_generic.html.haml

```diff
@@ -1,4 +1,8 @@
 = render :partial => "layouts/flash_msg"
+- if respond_to?(:flash_list)
+  - process_flash_messages(flash_list).each do |flash|
+    :javascript
+      add_flash("#{flash[:message]}", "#{flash[:level]}")
```

Comment:
single quotes + j_str to escape the strings,
or `miqFlashLater(flash.to_json)` + `miqFlashSaved()` :)

Reply:
Fixed.